tasks/ca/statcan/data.py

```diff
@@ -53,6 +53,22 @@ def download(self):
 
 
 class SplitAndTransposeData(BaseParams, Task):
+    INDIAN_RESERVES_SUFFIX = '-DQ'
```

Comment:
Nice to know what that suffix means. Where did you get it? I was looking for it and I was unable!